input_output.ipynb

```diff
@@ -1074,9 +1170,15 @@
    },
    "outputs": [],
    "source": [
-    "for i in range(0, 8**3):\n",
+    "for i in range(0, 6):\n",
+    "    bt = i.to_bytes(4)\n",
+    "    print(f\"bytes: {[hex(b) for b in bt]}, integer: {i}, string: {chr(i)}\")\n",
+    "for i in range(60, 74):\n",
+    "    bt = i.to_bytes(4)\n",
+    "    print(f\"bytes: {[hex(b) for b in bt]}, integer: {i}, string: {chr(i)}\")\n",
+    "for i in range(8**3, 8**3 + 3):\n",
     "    bt = i.to_bytes(4)\n",
-    "    print(f\"bytes: {bt}, integer: {i}, string: {chr(i)}\")"
+    "    print(f\"bytes: {[hex(b) for b in bt]}, integer: {i}, string: {chr(i)}\")"
```

Comment:
This cell didn't print any output for me

Reply:
Python version issue, thanks for catching that